protocol/messenger_mailserver.go

```diff
@@ -945,3 +945,53 @@ func (m *Messenger) ConnectionChanged(state connection.State) {
 
 	m.connectionState = state
 }
+
+func (m *Messenger) SyncChatOneMonth(chatID string) (uint32, error) {
```

Comment:
Maybe it would be better to call this just `SyncChat` and add a param for the time to sync? I just feel like if at one point our store nodes handle 2 months of data, then having to do a new function called `SyncChatTwoMonths` is a bit silly  haha.

Reply:
that's true :)